Rename Teachertool info page (#6064)
microsoft/pxt-microbit: docs/code-eval-tool.md

![Checklist name](/static/teachertool/checklist-name.png)

#### 3. Add Criteria

One or more **_criteria_** are added from the catalog using the **Add Criteria** button.
      
![Add Criteria](/static/teachertool/add-criteria.png)

Some criteria (like `[block] used [count] times`) can be added multiple times, others (like `Read a GPIO pin` can only be added once).

![Criteria items](/static/teachertool/criteria-items.png)

#### 4. Fill in Parameters

Parameters for the criteria item are filled in for a criteria item.

### ~ tip

#### Parameter types

From a technical perspective, criteria parameters have these types:

- **Numeric** parameters have a small input and only allow number inputs.
- **String** parameters can have medium and long sized inputs.
- **Block** parameters should open a block-picker modal.
- **Empty** parameters appear in an error state until they have values.

### ~

Here a block is selected and used 3 times:

![Criteria parameters 1](/static/teachertool/parameters-1.png)

Parameter options are displayed and then selected.

![Criteria parameters 2](/static/teachertool/parameters-2.png)
      
![Criteria parameters 3](/static/teachertool/parameters-3.png)

> 

Reviewer: Can we add a section here about AI?  suggestion:

#### 5. Ask AI

You can also Ask AI a question as a criteria in the checklist. You are limited to up to 5 Ask AI questions per checklist.

![Ask AI](/static/teachertool/ask-ai.png)

Author: No sweat, I got one.

#### 5. Remove Criteria

A criteria item is removed using the **trash** button.

![Remove Criteria](/static/teachertool/remove-criteria.png)

#### 6. Load a project

> A project into the project view by pasting in a share link or share ID.

Reviewer 2: ```suggestion
Load a project into the project view by pasting in a share link or share ID.
```

#### 2. Edit outcomes

An outcome is edited using the provided dropdown.
      
![Edit outcome](/static/teachertool/edit-outcome-1.png)

The new selected outcome.

![Edit outcome](/static/teachertool/edit-outcome-2.png)
      
### Result Clearing and Auto-Run

#### 1. Toggling Auto-run

Auto-run is toggled either **on** or **off** using the button in the menu.
      
![Auto-run button](/static/teachertool/autorun-button.png)

#### 2. Auto-run disabled

If auto-run is **disabled**, a result's outcome (i.e. "Looks good", "Needs work", etc...) is set to "Not started" automatically if any of any of the following conditions are met:
   
- It is newly added (defaults to the "Not Started" state).
- A parameter in a rule is changed (only the affected rule enters the "Not Started" state).
- The loaded project changes (all rules are be set to "Not started").

#### 3. Auto-run enabled

If auto-run is **enabled**, any rules that enter the "Not started" state due to the conditions listed above are immediately and automatically re-run with their results updated.

### Loading/Importing/Exporting Checklists

#### 1. Pre-built checklists

> There are pre-built checklists are available from on home page. If a selected checklist is already in-progress, an overwrite confirmation prompt is given.

Reviewer 2: ```suggestion
There are pre-built checklists are available on the welcome page. If a selected checklist is already in-progress, an overwrite confirmation prompt is given.
```

Two changes in this - one just a typo fix, but also modified home page to welcome page (which is how we refer to it later on in the document). I figured "welcome" is better since people might consider the "home page" to mean just microbit.makecode.com.

      
![Pre-built checklists](/static/teachertool/prebuilt-rubrics.png)
      
#### 2. Export a checklist

A checklist is exported using the vertical "..." menu near the "auto-run" button.

![Export checklist](/static/teachertool/export-checklist.png)

> This will download a json file for the checklist.

Reviewer 2: I wonder if we should add a note here specifying that only the checklist is exported, not results. For results, use "print".
